maelstrom

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# A small wrapper script for invoking the Maelstrom jar, with arguments.
+
+java -jar target/maelstrom-0.2.1-SNAPSHOT-standalone.jar "$@"
```

Comment:
Might I suggest using the version generated by the `package` script instead? Or `lein run`, if you're comfortable with Clojure.

Reply:
I ll use `lein run`. Since this is running from the sources, makes sense to use the latest sources instead of a compiled artefact which might go stale. Thoughts ?

Reply:
Yeah, lein run is probably best if hacking on Maelstrom itself, but if you're only changing your babashka scripts, either's fine. :-)